recipes/cfanalytics/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "cfanalytics" %}
+{% set version = "0.1.4" %}
+{% set sha256 = "aa3d98ebc1c39000c1e7ab08d586e079f8d7c87db15a318e03ff4918bc38c739" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - python
+    - pip
+  run:
+    - python 3.6
```

Comment:
you need python >=3.6

Reply:
appveyor seems to pick up python 2.7 but the others are fine. Guess I could use a flag to switch off the windows build but would be nice to know why it's not picking up just the python 3.6 build